config/locales/en/course/assessment/answer/answers.yml

```diff
@@ -2,7 +2,8 @@ en:
   course:
     assessment:
       answer:
-        error: 'An error occurred while evaluating your answer, please try again.'
+        error: >
+          Sorry. Looks like something went wrong on our end, please try again.
```

Comment:
Ohh. "Sorry, the autograder is having mood swings and quit on us. Please try again."